src/DIRAC/Interfaces/scripts/dput.py

```diff
@@ -111,9 +111,11 @@ def getRecursive(self):
         # use "NO DEFAULT" to distinguish no default SE set from all
         # other error cases
         if not retVal["OK"]:
-            print("Error:", retVal["Message"])
+            gLogger.error("Error:", retVal["Message"])
```

Comment:
This a very general comment, applicable in many places in this PR:

```suggestion
            gLogger.error(retVal["Message"])
```
because of course there's no need to repeat "Error". 

This should be, I believe, part of changing from `print` to `gLogger`, so apply in this PR already.